packages/riipen-ui/src/components/Tab.test.jsx

```diff
@@ -0,0 +1,363 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPhone } from "@fortawesome/free-solid-svg-icons";
+
+const icon = i => props => <FontAwesomeIcon icon={i} {...props} />;
+
+import Tab from "./Tab";
+
+describe("<Tab>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Tab value />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Tab value />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Tab value />);
+
+      expect(wrapper.find("Tab").props().active).toEqual(false);
+      expect(wrapper.find("Tab").props().breakpoint).toEqual("sm");
+      expect(wrapper.find("Tab").props().color).toEqual("secondary");
+      expect(wrapper.find("Tab").props().disabled).toEqual(false);
+      expect(wrapper.find("Tab").props().fullWidth).toEqual(false);
+      expect(wrapper.find("Tab").props().orientation).toEqual("horizontal");
+    });
+  });
+
+  describe("active and displayActive props", () => {
```

Comment:
There's a lot going on in this set of tests which I think we could simplify or spread out a bit. A general rule of writing tests is "don't make the next person think" when they have to read / update your tests, and I had to do more thinking than usual with these ones.

There couple things that are going on here:

1. We want to make sure that the correct class name, based on color, is set when active is true
2. We want to make sure that the correct class name, based on color, is set when displayActive is true
3. We want to make sure that the correct class name, based on orientation, is set when active is true
4. We want to make sure that the correct class name, based on orientation, is set when displayActive is true

Those being our cases, I think we can split this one giant describe block out into each prop's unique describe block.

- Case 1 above could be part of the `active` prop tests.
- Case 2 above could be part of the `displayActive` prop tests.
- Case 3 above could be part of the `active` prop tests.
- Case 4 above could be part of the `displayActive` prop tests.

We can also simplify the tests in general:

- We probably don't need to test every single color / orientation, just one is probably good enough for each case I just described.
- We don't need to test `active` and `displayActive` in tandum with each other since they actually don't affect each other